packages/liblc3/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+pkgname=liblc3
+pkgver=1.0.1
+pkgrel=1
+pkgdesc='Low Complexity Communication Codec library and tools'
+arch=(aarch64 x86_64)
+url='https://github.com/google/liblc3/'
+license=(Apache)
+groups=(pro-audio)
+depends=(glibc)
+makedepends=(meson)
+provides=(liblc3.so)
```

Comment:
Shouldn't this go into the `package` function? I'm not sure if it makes a difference technically, but I think it's better to be consistent, isn't it?

Reply:
I've only ever seen that (`provides` inside `package`) for split packages, otherwise it's always set globally.

Reply:
I thought I saw that with provides for libs somewhere, but now I can't find it. Never mind then.